### PR TITLE
feat(24.04): add erlang slices

### PR DESCRIPTION
**Pull request by linostar**
# Proposed changes

Add erlang-* slices to 24.04

Note that erlang packages are not available for i386 arch.

I tried to add tests for all erlang binaries , but some of them needs the erl server to be running in the background, and attempting to run it with `&` or with `nohup` has failed in the spread container (although it works locally).

## Related issues/PRs

PR #258

## Forward porting

N/A

## Checklist

* [x] I have read the [contributing guidelines](\nhttps://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have tested my changes ([see how](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md#7-test-your-slices-before-opening-a-pr))
* [x] I have already submitted the [CLA form](\nhttps://ubuntu.com/legal/contributors/agreement)

## Additional Context

N/A

### CODE REVIEW

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
Following up from https://github.com/canonical/chisel-releases/pull/384#discussion_r1840788785

> bash is needed because there are several erlang packages that have shell scripts, and they all depend on erlang-base, so it makes sense to put bash as dependency in erlang-base.
> coreutils is needed because some bash scripts use /usr/bin/env to detect the path of escript, and env is found in coreutils
> as for libtinfo8t64, I found that some of the binaries in erlnag-snmp are trying to open libtinfo so file (from strace output), and I am not sure if there are others that use it as well, so I put it in erlang-base's essential as a precaution.

What are the several erlang packages that have shell scripts? If those are other than the `erlang-base`, then those should be the ones carrying this `bash` dependency. 

For `coreutils`, same as for `bash`. Additionally, if all you need is `env`, then let's not bring in all the binaries.

And `libtinfo8t64`, adding it just as a precaution isn't a good practice cause we may be unnecessarily bloating the slice, and once introduced, it's not something we can delete (not at least until the following Ubuntu release).

File: slices/coreutils.yaml [hunk 1]
```diff
@@ -129,6 +129,12 @@ slices:
       /usr/bin/yes:
       /usr/sbin/chroot:
 
+  env:
```
Comment (cjdcordeiro) [inline slices/coreutils.yaml:132]: since this slice is being introduced and it's a binary, can you please add a test for it?

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
LGTM. just missing a test and that would be it

**Review by zhijie-yang — APPROVED** [review]
LGTM, thanks!

**Pull request merged**